### ISSUE TITLE
Не строится сводный приказ

### ISSUE THREAD

**Issue by gsbelarus** [GoldenSoftwareLtd/gedemin#1955]
Originally reported on Google Code with ID 1955

```
На СлоМК ставили эталонные настройки. Сводный приказ не строится.
```

Reported by `gs1994` on 2010-04-14 14:12:20

**Comment by gsbelarus** [comment]
```
СлоМК - это что? На БД slonim он строится
```

Reported by `alexandra.gsoftware` on 2010-04-14 14:52:52

**Comment by gsbelarus** [comment]
Reported by `alexandra.gsoftware` on 2010-04-17 15:22:56
- **Status changed**: `Duplicate`
- **Merged into**: #1891
